vagrant/arm/vagrantfile

```diff
@@ -0,0 +1,42 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# DO NOT REMOVE: Monkey patch (https://github.com/hashicorp/vagrant/issues/8878) to fix DHCP error.
+class VagrantPlugins::ProviderVirtualBox::Action::Network
+  def dhcp_server_matches_config?(dhcp_server, config)
+    true
+  end
+end
+
+# Number of worker nodes
+WORKER_NODES = 2
+
+Vagrant.configure("2") do |config|
+  # Global config
+  config.vm.box = "bento/ubuntu-24.04"
+  config.vm.box_version = "202404.26.0"
+  config.vm.network "private_network", type: "dhcp"
```

Comment:
Great that each VM has a private network! Have you also checked each VM can "directly communicate with all other VMs"?

Reply:
Ah nice, could you perhaps add a section under the README with instructions on how to do this? (and maybe add screenshot/example)